apps/src/studio/blocks.js

```diff
@@ -572,7 +572,7 @@ exports.install = function(blockly, blockInstallOptions) {
         .appendTitle(new blockly.FieldDropdown(skin.itemChoices), 'CLASS');
 
       var dropdown = new blockly.FieldDropdown(this.VALUES);
-      dropdown.setValue(this.VALUES[2][1]); // default to slow
+      dropdown.setValue(this.VALUES[1][1]); // default to slow
```

Comment:
It is stated as "slow" in the UI, but VERY_SLOW in absolute terms. Basically, slow/normal/fast for characters/items are defined to be one step slower than slow/normal/fast for bot speed.